controllers/runner_controller.go

```diff
@@ -171,15 +171,15 @@ func (r *RunnerReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 		}
 
 		restart := false
-
-		if pod.Status.Phase == corev1.PodRunning {
+		if pod.Status.Phase == corev1.PodFailed {
 			for _, status := range pod.Status.ContainerStatuses {
 				if status.Name != containerName {
 					continue
 				}
 
-				if status.State.Terminated != nil && status.State.Terminated.ExitCode == 0 {
+				if status.State.Terminated != nil && status.State.Terminated.ExitCode == 130 {
```

Comment:
Can we have a reference to any doc/sentence that helps making the sense of this number `130`?
I did a quick googling for `systemd exit status` and `systemd 130` but had no luck. So perhaps this is a different thing?

Reply:
To be honest, I'm not sure about that either. systemd is always started as PID 1, so theres no way to know systemd's exit code without reading the code. Exit code `130` is based on the pod status after executing `kill 37 1` in the Pod.